test/ssl-tests/13-fragmentation.conf.in

```diff
@@ -184,4 +184,85 @@ our @tests = (
             MaxFragmentSize => 4 * 1024,
         }
     },
+    ####                          #######
+    # Default fragment size is 512.
+    # Default Application data size is 256.
+    {
+        name => "Maximum Fragment Len extension set to 512",
+        server => { },
+        client => {
+            extra => {
+                MaxFragmentLenExt => 512,
+            },
+        },
+        test => {
+            ApplicationData => 1024,
+            MaxFragmentSize => 1024,
```

Comment:
I don't understand the point of setting MaxFragmentSize here (and in the other tests below). It seems to set the max_send_fragment value. Unless you are testing the interaction between max_fragment_len and max_send_fragment?